docs/input/docs/installation/obtain.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
I would move this under the Getting Started section

Reply:
Moved